plugins/http_extract/src/http_extract/component.py

```
@@ -0,0 +1,580 @@
+        """Генерация ключа для кэша."""
+        params_str = json.dumps(params or {}, sort_keys=True)
+        cache_string = f"{url}:{params_str}"
+        return hashlib.sha256(cache_string.encode()).hexdigest()
+
+    @classmethod
+    def process_info(cls) -> Info:
+        return Info(
+            name="HTTPExtract",
+            version="1.0.0",
+            description="Извлечение данных из REST API\
+            с поддержкой аутентификации и пагинации",
```

Ruff (0.12.2)

String contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF001)